inc/cachehandlers/disk.php

```diff
@@ -33,24 +32,16 @@ function connect($silent=false)
 	 * Retrieve an item from the cache.
 	 *
 	 * @param string $name The name of the cache
-	 * @param boolean $hard_refresh True if we should do a hard refresh
 	 * @return mixed Cache data if successful, false if failure
 	 */
-	function fetch($name, $hard_refresh=false)
+	function fetch($name)
 	{
 		if(!@file_exists(MYBB_ROOT."/cache/{$name}.php"))
 		{
 			return false;
 		}
 
-		if(!isset($this->cache[$name]) || $hard_refresh == true)
-		{
-			@include(MYBB_ROOT."/cache/{$name}.php");
-		}
-		else
-		{
-			@include_once(MYBB_ROOT."/cache/{$name}.php");
-		}
+		@include(MYBB_ROOT."/cache/{$name}.php");
```

Comment:
I'm not 100% sure how the disk cache work. However it seems to save the variable as php variable. And an easy example of how to break it would be to save a cache named `name` (though that would likely break anyways). Just used that as it was always used till now (the variable in the if was never set as it doesn't exist).